interface/wx/webview.h

```diff
@@ -460,11 +460,45 @@ class wxWebView : public wxControl
     virtual void Reload(wxWebViewReloadFlags flags = wxWEBVIEW_RELOAD_DEFAULT) = 0;
 
     /**
-        Runs the given javascript code.
-        @note When using wxWEBVIEW_BACKEND_IE you must wait for the current
+        Sets IE emulation level.
```

Comment:
You should explain why would you want/need to call this function, even if only briefly -- and also add something like `@see RunScript() for further details`.